lib/Dialect/LinalgTransform/IR/LinalgTransformOps.cpp

```diff
@@ -699,17 +699,23 @@ transform::LowerVectorsOp::apply(transform::TransformResults &results,
 // BufferizeOp
 //===---------------------------------------------------------------------===//
 
+static void applyBufferizationEnablingTransformations(ModuleOp moduleOp) {
+  RewritePatternSet patterns(moduleOp.getContext());
+  patterns.add<GeneralizePadOpPattern>(moduleOp.getContext());
+  (void)applyPatternsAndFoldGreedily(moduleOp, std::move(patterns));
```

Comment:
Propagate failure maybe?

Reply:
This helper comes from comprehensive bufferization pass where the failure wasn't propagated either. @matthias-springer is that intentional? Do we want to propagate failures here?

Reply:
I'm keeping this as is for now, not to change the behavior compared to prior nested PassManager-based implementation. We can amend this change later if it's safe to propagate failures.